NEWS.md

```diff
@@ -390,7 +390,7 @@ Outstanding ones only.
 
     * Improved support for YAML [[Bug #8382]]
 
-    * Use officially discouraged. Read "Caveats" section.
+    * Use officially discouraged. Read "Caveats" section in the API documentation.
```

Comment:
"The API documentation" feels still ambiguous.

```suggestion
    * Use officially discouraged. Read "rdoc-ref:OpenStruct@Caveats"
      section.
```

Reply:
@nobu Thank you for your suggestion. I updated it.

Reply:
Found that RDoc includes double-quote and dot in the anchor text.
You may want to remove these punctuations or use `["Caveats" section](rdoc-ref:OpenStruct@Caveats).`

Reply:
@nobu I updated again. Hope it works.